components/service/glean/src/main/java/mozilla/components/service/glean/EventMetricType.kt

```diff
@@ -24,12 +24,13 @@ internal const val MAX_LENGTH_OBJECT_ID = 20
  * data and making sure that limits are enforced.
  */
 data class EventMetricType(
-    override val applicationProperty: Boolean,
-    override val disabled: Boolean,
-    override val group: String,
+    override val applicationProperty: Boolean = false,
+    override val disabled: Boolean = false,
+    override val category: String,
     override val name: String,
-    override val sendInPings: List<String>,
-    override val userProperty: Boolean,
+    override val sendInPings: List<String> = listOf("default"),
+    override val userProperty: Boolean = false,
+    val objects: List<String>,
```

Comment:
Darn! You just found a bug in my implementation of events. `objectId` from the `record()` function of the EventMetricType must be in the list of `objects`. We can either fix it here or we can file a bug and fix it later, up to you!

Reply:
I can fix it here.  Good chance for me to learn more Kotlin.